deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -59,6 +59,13 @@ PLATFORM_CONFIG=F110MHz_BaseF1Config
 instancetype=z1d.2xlarge
 deploytriplet=None
 
+# Single-core, Rocket-based recipes with Small NVDLA
+[firesim-rocket-singlecore-small-nvdla-no-nic-l2-llc4mb-ddr3]
+DESIGN=FireSim
+TARGET_CONFIG=WithNVDLASmall_DDR3FRFCFSLLC4MB_FireSimGemminiRocketConfig
```

Comment:
Just curious, why the `Gemmini` config?

Reply:
We're going to move this out too.

Reply:
Ignore that, i misunderstood the question.

Reply:
Typo. Fixed.